docs/reference/export.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: default
+title: Exporting data
+description: Use lakeFS spark client to export lakeFS commit to the object store. 
+parent: Reference
+nav_exclude: true
+has_children: false
+---
+
+# Exporting data
+{: .no_toc }
+The export operation copies all data from a given lakeFS commit to 
+a designated object store location.
+
+For instance, the contents `lakefs://example@master` might be stored on
```

Comment:
Consider:

```suggestion
For instance, the contents `lakefs://example@master` might be exported on
```

("stored" implies to me that the actual files are there all the time)

Reply:
Done.